src/main/java/seedu/address/logic/commands/OptionCommand.java

```diff
@@ -0,0 +1,74 @@
+package seedu.address.logic.commands;
+
+import java.nio.file.InvalidPathException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import seedu.address.commons.exceptions.IllegalValueException;
+import seedu.address.commons.util.CollectionUtil;
+import seedu.address.model.Model;
+import seedu.address.model.UserPrefs;
+
+/**
+ *
+ * Command to change user preference.
+ *
+ */
+//@@author A0126539Y
+public class OptionCommand extends Command{
+    public static final String COMMAND_WORD = "option";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Change user preference. "
+            + "Parameters: [save/Save Location]\n"
+            + "Example: " + COMMAND_WORD
+            + " save/myFileName.xml";
+
+    public static final String MESSAGE_SUCCESS_PREFIX = "Preference saved: ";
+    public static final String MESSAGE_SAVE_LOCATION_SUCCESS = "Save Location";
```

Comment:
We might have several other options to change, it would be too cluttered I think.